xclim/indices/_conversion.py

```diff
@@ -74,7 +78,7 @@ def uas_vas_2_sfcwind(
 
     if not return_direction:
         wind.attrs["units"] = "m s-1"
-        return wind
+        return wind, None
```

Comment:
@aulemahal This is a change that is potentially breaking, so I'm curious as to what you think here: Depending on inputs to the function, the number of outputs will change. I've come to understand that this isn't the best practice to follow. As such, I figured it would make sense to always return two outputs (`None` or `xr.DataArray`), but this requires a fix/tests to the Indicator class `atmos.wind_speed_from_vector`.

Is there a better way to organize this? Would it make more sense to split these outputs into two separate functions? I need development/design advice.

Reply:
Oh! That's one change I overlooked when pushing the PR on multiindicators! The `return_direction` is not necessary anymore. I can push a fix today (in a separate PR, its cleaner), so that the function always return 2 DataArrays.

Reply:
Awesome. Thanks!